todo_app/data/trello_items.py

```diff
@@ -94,4 +94,33 @@ def complete_item(card_id):
 
     response = requests.request("PUT", reqUrl, params=query_params)
     
+    print(response.text)
+def doing_item(card_id):
+
+
+    reqUrl = f"https://api.trello.com/1/cards/{card_id}/"
+
+    query_params = {
+        "key": trello_key,
+        "token": trello_token,
+        "idList": "626653880ecf1c46be8073f9"
+    }
+
+    response = requests.request("PUT", reqUrl, params=query_params)
+    
+    print(response.text)
```

Comment:
I like the name of this function - though it isn't actually used by the "doing" button just yet! I'd check that you have the right `idList` here and that you're calling this from `/items/<id>/doing` endpoint.